eng/tools/suppressions/test/e2e.test.ts

```diff
@@ -0,0 +1,38 @@
+import { expect, test } from "vitest";
```

Comment:
How about renaming `e2e.test.ts` to `suppressions_e2e.test.ts` or similar? I have slight preference against having multiple files with the same name - it makes navigating in IDE slightly harder.

Reply:
Renamed to `suppressions.test.e2e.ts`